config/config.yaml.example

```diff
@@ -100,6 +100,6 @@ github:
     events: ["*"]
 
 events:
-  driver: go-channel
+  driver: postgresql
```

Comment:
If we're changing the defaults (I'm not sure if we should), we should also add defaults for the database configuration. I actually suggest not to change these defaults just yet.